content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -834,11 +834,34 @@ commands.spawn((
 ));
 ```
 
-## Exposure Settings
+## Camera Exposure
 
-<div class="release-feature-authors">authors: @TODO</div>
+<div class="release-feature-authors">authors: @superdump (Rob Swain), @JMS55, @cart</div>
 
-TODO.
+Cameras in the "real world" control how bright an image is by adjusting "exposure settings":
```

Comment:
I'd like to see a bit of a shout out to the whole idea of PBR: why are we trying to emulate real cameras at all?